src/main/java/com/ip/tasksmanager/Duke.java

```diff
@@ -0,0 +1,144 @@
+package com.ip.tasksmanager;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    public static String line = "____________________________________________________________";
+    public static final int INPUT_PHRASES_COUNT = 2;
+
+    public static Tasks[] tasks = new Tasks[100]; //for storing (all types of) tasks
```

Comment:
I like the fact that you wrote clear comments for some variables and all methods used. :D